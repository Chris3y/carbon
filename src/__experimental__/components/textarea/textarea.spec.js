import React from 'react';
import { shallow } from 'enzyme';
import Textarea from '.';

describe('Textarea', () => {
  it('renders with InputPresentation and Input and all props passed to Input', () => {
    const wrapper = shallow(
      <Textarea value='foobar' />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
