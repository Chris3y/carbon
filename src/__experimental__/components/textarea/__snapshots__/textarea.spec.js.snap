// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea renders with InputPresentation and Input and all props passed to Input 1`] = `
<FormField
  disabled={false}
  enforceCharacterLimit={true}
  expandable={false}
  readOnly={false}
  size="medium"
  value="foobar"
  warnOverLimit={false}
>
  <styled.div>
    <InputPresentation
      disabled={false}
      enforceCharacterLimit={true}
      expandable={false}
      readOnly={false}
      type="text"
      value="foobar"
      warnOverLimit={false}
    >
      <TextareaInput
        disabled={false}
        inputRef={[Function]}
        readOnly={false}
      />
    </InputPresentation>
  </styled.div>
</FormField>
`;
